src/run_tracker_interactive.py

```diff
@@ -9,16 +9,30 @@
 
 DATASET_PATH = "../data/Mini-OTB"
 
+def _scale_factor(a):
+    amin = np.min(a)
+    amax = np.max(a)
+    if amax-amin <= 1e-3: 
+        scale_factor = 1.1*amax
+    else:
+        scale_factor = amax-amin
+    return scale_factor
+
 if __name__ == "__main__":
     import argparse
     parser = argparse.ArgumentParser(conflict_handler="resolve")
     parser.add_argument("--seq-idx", type=int, default=0)
     parser.add_argument("--bbox-enlarge-factor", type=float, default=1.0)
+    parser.add_argument("--visualize-filter", type=bool, default=False)
     trackers.add_tracker_args(parser)
     args = parser.parse_args()
 
+    visualize_filter = args.visualize_filter
     tracker = trackers.get_tracker(args)
 
+    if visualize_filter: 
```

Comment:
I visualized the grayscale filters in my notebook here FYI: https://github.com/TobiasNorlund/lfr-module-2/blob/master/notebooks/MOSSE.ipynb